docs/systemd.md

```diff
@@ -58,16 +58,24 @@ dinamically, you can find the remaining options needed in
 
 ## Systemd units on WSL
 
-`systemd` is supported on WSL starting from version `0.67.6` and higher. Note that you need to have Windows 11 to install this version.
+`systemd` is supported on WSL starting from version `0.67.6` and higher. You can check your version by running `wsl --version` command.
```

Comment:
```suggestion
`systemd` is supported on WSL starting from version `0.67.6` and higher.
You can check your version by running the `wsl --version` command.
```

Reply:
Oh my suggestion was probably confusing, I see that you took the newline, but I also added a "the" here